code/src/App.js

```diff
@@ -1,9 +1,48 @@
 import React from 'react'
+import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom'
+
+import MovieList from './pages/MovieList'
+import MovieDetails from 'pages/MovieDetails'
+import Cast from './pages/Cast'
+import Crew from './pages/Crew'
+import PageNotFound from './pages/PageNotFound'
+import MovieProjectsByCast from './pages/MovieProjectsByCast'
+import MovieProjectsByCrew from './pages/MovieProjectsByCrew'
+
 
 export const App = () => {
+
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <Switch>
+        <Route path="/" exact>
+          <MovieList />
+        </Route>
+        <Route path="/movies" exact>
+          <Redirect to="/" />
+        </Route>
+        <Route path="/movies/:movieId" exact>
+          <MovieDetails />
+        </Route>
+        <Route path="/movies/:movieId/cast" exact>
+          <Cast />
+        </Route>
+        <Route path="/profile/cast/:castId">
+          <MovieProjectsByCast />
+        </Route>
+        <Route path="/movies/:movieId/crew" exact>
+          <Crew />
+        </Route>
+        <Route path="/profile/crew/:crewId">
+          <MovieProjectsByCrew />
```

Comment:
Perhaps Cast and Crew could be the same component and you just send in different props? If you'd like to clean up the code a bit 😉 🥰

Reply:
Yeah, i guess so. They are kind of repetitive.